README.md

```diff
@@ -1,16 +1,158 @@
+
+  #### Serving
+
+  You can choose from the following options:
+
+  - OpenFunction Serving implementation based on Knative, you need to [install the Knative project](https://knative.dev/docs/install/).
+
+  ## CustomResourceDefinitions
+
+  ---
+
+  The core functionality of OpenFunction is to serve users with the ability to develop, run, and manage business applications as execution units of code functions. OpenFunction is implemented based on the following [custom resource definitions (CRDs)](https://kubernetes.io/docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions/):
+
+  - **```Function```**, abstract definition of a function.
+  - **```Builder```**, abstract definition of a function builder.
+  - **```Serving```**, abstract definition of a function workload.
+
+  ## QuickStart
+
+  ---
+
+  ### Install
+
+  You can quickly build an OpenFunction platform by executing the following command after selecting an appropriate version.
+
+  ```shell
+  kubectl apply -f config/bundle.yaml
+  ```
+
+  > Note: When using non-default namespaces, make sure that the ClusterRoleBinding in the namespace is adapted.
+
+  ### Sample: Run a function.
+
+  If you have already built implementations of OpenFunction Builder and OpenFunction Serving, follow steps below to run a simple case.
+
+  1. Creating a secret
+
+     In order to access your mirror repository, you need to create a secret. You can create this secret by editing the ``username`` and ``password`` fields in ``config/samples/registry-account.yaml``, and then apply it.
+
+     ```shell
+     kubectl apply -f config/samples/registry-account.yaml
+     ```
+
+  2. Creating functions
+
+     Here is a simple Function case for you, modify the ``spec.image`` field in ``config/samples/core_v1alpha1_function.yaml`` to your own repository address: 
+
+     ```yaml
+     apiVersion: core.openfunction.io/v1alpha1
+     kind: Function
+     metadata:
+       name: function-sample
+     spec:
+       image: "<your registry name>/sample-go-func"
+     ```
+
+     Use the following command to create this Function:
+
+     ```shell
+     kubectl apply -f config/samples/core_v1alpha1_function.yaml
+     ```
+
+  3. Result observation
+
+     You can observe the process of Function with the following command:
+
+     ```shell
+     kubectl get functions.core.openfunction.io
+     
+     NAME              AGE
+     function-sample   5s
+     ```
+
+     You can also observe the process of Builder in the [Tekton Dashboard](https://tekton.dev/docs/dashboard/).
+
+     Finally, you can observe the final state of the case in the Serving:
+
+     ```shell
+     kubectl get servings.core.openfunction.io
+     
+     NAME                      AGE
+     function-sample-serving   15s
+     ```
+
+     At this point, you can view the service entry for the case function with the following command:
```

Comment:
=>
You can now find out the service entry of the function with the following command: